draft-ietf-jsonpath-base.md

```diff
@@ -1063,27 +1063,27 @@ descendant-selector = ".." ( dot-member-name      /  ; ..<name>
                            )
 ~~~~
 
-Note that there is no "bald" descendant selector `..`.
+Note that there is no "bald" descendant selector (`..` on its own).
 
 #### Semantics
 {: unnumbered}
 
-The `descendant-selector` selects certain descendants of a node:
+A `descendant-selector` selects certain descendants of a node:
```

Comment:
Does `A` Indicate `any` or `for every`?

Reply:
I changed "The" to "A" because there are multiple descendant selectors. It carries the connotation of "any".